### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by DemiMarie-parity**
✄ -----------------------------------------------------------------------------

### CODE REVIEW

**Comment by bkchr** [comment]
Could you please do this next time just in one pr? There is no reason to open 2 prs for this.

**Comment by DemiMarie-parity** [comment]
> Could you please do this next time just in one pr? There is no reason to open 2 prs for this.

@bkchr Sorry about that.  I use the `hub` command-line tool to create PRs, and I didn’t realize the first one had already been merged.

File: Cargo.toml [hunk 1]
```diff
@@ -126,3 +126,6 @@ is-it-maintained-open-issues = { repository = "paritytech/substrate" }
 [profile.release]
 # Substrate runtime requires unwinding.
 panic = "unwind"
+
+[patch.crates-io]
+tiny-bip39 = { git = "https://github.com/paritytech/tiny-bip39" }
```
Comment (bkchr) [inline Cargo.toml]: Needs to be removed.

**Comment by kianenigma** [comment]
interesting to use in the future 
https://blog.rust-lang.org/inside-rust/2019/10/03/Keeping-secure-with-cargo-audit-0.9.html

**Pull request merged**